client/src/Features/PublishAssignment/PublishActionButtons.tsx

```diff
@@ -1,151 +1,159 @@
-import React, { useState } from 'react';

```

Comment:
It's very difficult to figure out what exactly has been changed in this file. Has this whole file changed?
If not, can we somehow change it to only highlight your changes in it?

Reply:
Sorry about this, I think the prettier just updated the complete file. I am reverting this from my end and that should help.

Reply:
Done! It should be visibly clear now.